components/developer/swig/Makefile

```diff
@@ -85,11 +89,12 @@ REQUIRED_PACKAGES += runtime/java/openjdk11
 REQUIRED_PACKAGES += developer/parser/bison
 REQUIRED_PACKAGES += runtime/lua
```

Comment:
Something is wrong with this dependency. We don't have this package anymore:
~$ pkg list -af|grep bison
SUNWbison                                         2.3-0.133                  --r
developer/parser/bison                            3.8.2-2020.0.1.1           --r
developer/parser/bison-37                         3.7.6-2020.0.1.0           i--
developer/parser/bison-38                         3.8.2-2020.0.1.1           i--
system/library/bison-runtime                      3.8.2-2020.0.1.1           i--